scripts/jenkins/jenkinsfiles/Jenkinsfile-Release

```diff
@@ -180,6 +180,36 @@ try {
                         }
                     }
                 }
+                
+             def HELM_STAGE_NAME = 'Publish HELM to S3'
+                stage(HELM_STAGE_NAME) {
+                    try {
+                        pipelineContext.getBuildSummary().addStageSummary(this, HELM_STAGE_NAME, env.BUILD_NUMBER_DIR)
+                        pipelineContext.getBuildSummary().setStageDetails(this, HELM_STAGE_NAME, env.NODE_NAME, env.WORKSPACE)
+                        // Helm automatically installs latest chart if version is unspecified by the user, therefore there is no need for a latest tag
+                        insideDocker([], pipelineContext.getBuildConfig().AWSCLI_IMAGE, pipelineContext.getBuildConfig().DOCKER_REGISTRY, pipelineContext.getBuildConfig(), 3, 'HOURS') {
+                            if (!params.TEST_RELEASE) {
+                                sh """
+                                    cd ${env.BUILD_NUMBER_DIR}/h2o-helm/
+                                    sed -i 's/version: [0-9]*\.[0-9]*\.[0-9]*/version: ${env.PROJECT_VERSION}/' Chart.yaml
+                                    sed -i 's/appVersion: [0-9]*\.[0-9]*\.[0-9]*/version: ${env.PROJECT_VERSION}/' Chart.yaml
+                                    helm package .
+                                    mkdir h2o-3
+                                    cp h2o-${env.PROJECT_VERSION}.tgz h2o-3/
```

Comment:
should this be eg. h2o-3/nightly vs h2o-3/stable?

Reply:
That's one way to deal with nightly vs stable - to have multiple folders. Could be.

Reply:
We'll have two types of charts then - with two different names - the actuallly difference is made in the chart name in `Chart.yaml`, the folders are only relevant for `index.yaml` in the repository itself.

Reply:
I'm not sure we want nightlies. This would bloat the repository index with lots of entries. If someone wants nightly H2O-3, they have the chance to do that with the existing helm chart - just change the tag name. A chart is not specifically tied to a single H2O version.

Reply:
ok, I thought would be good to have to test the release process but OK

Reply:
Resolved by having extra Helm repo just for nightlies.